### PR TITLE
Update addon-manager to use debian-base:v1.0.0

### PR DESCRIPTION
**Pull request by MrHohn**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Rebasing addon-manager on debian-base:v1.0.0 to pick up some CVE fixes (picking up #75678).

The manifest update will come as a follow-up.

**Which issue(s) this PR fixes**:

Fixes #NONE

**Special notes for your reviewer**:
/assigm @tallclair 
cc @jingyih

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jingyih** [comment]
Thanks @MrHohn!

File: cluster/addons/addon-manager/Makefile [hunk 1]
```diff
@@ -15,10 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v9.0
+VERSION=v9.1
```
Comment (tallclair) [inline cluster/addons/addon-manager/Makefile]: We usually recommend `v9.0.1` when nothing is changing except the base package versions. Up to you.

**Review by tallclair — COMMENTED** [review]
Thanks.
/lgtm
/approve

File: cluster/addons/addon-manager/Makefile (hunk 1 above)
Comment (MrHohn) [reply cluster/addons/addon-manager/Makefile]: Thanks Tim, that is better. Updated.

**Comment by MrHohn** [comment]
/assign @mikedanese 
for approval.

**Comment by MrHohn** [comment]
Try to approve this :)
/approve

**Pull request merged**